packages/myst-transforms/src/frontmatter.ts

```diff
@@ -20,7 +19,7 @@ type Options = {
 export function getFrontmatter(
   file: VFile,
   tree: GenericParent,
-  opts: Options = { removeYaml: true, removeHeading: true, propagateTargets: true },
+  opts: Options = { propagateTargets: true },
```

Comment:
I took out some of these options at the code level - we were never passing in `false` anyway. Now, we _always_ remove yaml frontmatter block, and heading removal only happens if it is a top-level h1 and the user did not explicitly set `content_includes_title: true`.